lib/elixir/src/elixir_import.erl

```diff
@@ -160,7 +160,7 @@ ensure_keyword_list(Meta, File, _Other, Kind) ->
 %% ERROR HANDLING
 
 format_error({invalid_import, {Receiver, Name, Arity}}) ->
-  io_lib:format("cannot import ~ts.~ts/~B because it doesn't exist",
+  io_lib:format("cannot import ~ts.~ts/~B because it doesn't exist or is declared as private",
```

Comment:
what if we say "because it is undefined or private"? Following the convention we have on UndefinedFunctionError?